src/System.CommandLine/Parsing/ParseOperation.cs

```diff
@@ -319,11 +322,26 @@ void ParseDirective()
                     result.AddValue(withoutBrackets.Slice(indexOfColon + 1).ToString());
                 }
 
-                _action = directive.Action;
+                if (directive.Action is not null)
+                {
+                    if (directive.Action.Exclusive)
+                    {
+                        _primaryAction = directive.Action;
+                    }
+                    else 
+                    {
+                        if (_passThroughActions is null)
+                        {
+                            _passThroughActions = new();
+                        }
+
+                        _passThroughActions.Add(directive.Action);
```

Comment:
nit: we can save some space here

```suggestion
                        (_passThroughActions ??= new()).Add(directive.Action);
```